Composite/Composite.Cs.Tests/Composites/ForkTests.cs

```diff
@@ -21,9 +17,91 @@ public void TransformationTest()
                 C.Value( new Simple { Number = 6, } ),
             });
 
-            var result = obj.Fork(x => x.Number == 1, _ => new[]{new Simple { Number = 4, }, new Simple { Number = 5, }});
+            var result = obj.Fork(x => x.Number == 1, _ => new[] { new Simple { Number = 4, }, new Simple { Number = 5, } });
 
             Assert.Equal("[ [ 4, 5 ], 6 ]", result.AsString(x => x.Number.ToString()));
```

Comment:
The `AsString()` seems to be a valuable `Composite<T>` extension. Suggested moving it to the library. The same goes for  `AsLimited()`